### PR TITLE
DS-777 STANFORD-422 | Add FontAwesome React package and create Multicolumn Fact Panel component; update Decanter

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Install FontAwesome React latest version with the free regular and solid icons
- Update Decanter
- Create FAIcon component which makes all the regular/solid free icons available for use
- Create Multicolumn Fact Panel component

# Review By (Date)
- Soon would be great so I can use the FA icons for other tickets

# Criticality
- 6

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to https://news.stanford.edu/developreport/yvonne-test-standard-page
2. Test the fact panel and in dark mode. See the layout with both 3 and 2 facts.
3. Check all responsive behavior.
4. Look at code

# Associated Issues and/or People
- JIRA ticket(s) - STANFORD-422

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "autoprefixer": "^10.4.16",
     "chalk": "^5.3.0",
     "core-js": "^3.33.3",
-    "decanter": "^7.2.0",
+    "decanter": "^7.3.0",
```
Comment (yvonnetangsu) [inline package.json:36]: Updated Decanter from 7.2.0 to 7.3.0

File: components/multicolumn-fact-panel/manifest.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              "2",
+              "3",
+              "4",
+              "5",
+              "6",
+              "7",
+              "8",
+              "9",
+              "10"
+            ]
```
Comment (yvonnetangsu) [inline components/multicolumn-fact-panel/manifest.json:86]: Reusing code from the Single Text Block PR which allows content author to add the amount of padding they want.

**Comment by yvonnetangsu** [comment]
@iamrentman @broleomo I just noticed that my styles are gone from the DEV preview 🤔 After I updated the DEV git bridge last Friday, the styles were there. Please hold off on reviewing for now.

**Comment by yvonnetangsu** [comment]
@broleomo @iamrentman This is ready for your eyes - no rush :). I added an example with just 1 fact at the bottom.

File: components/multicolumn-fact-panel/Component.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        </h2>
+      )}
+      {!!facts?.length && (
+        <div className="su-flex su-flex-col su-items-stretch lg:su-flex-row su-gap-34 md:su-gap-61 lg:su-gap-0 lg:su-mt-162 2xl:su-mt-171 su-divide-y-2 lg:su-divide-y-0 lg:su-divide-x-2 su-divide-black-30 dark:su-divide-black-60">
+          {facts.map((fact, index) => (
+            <div
+              key={fact.icon}
+              className="su-flex su-flex-col su-items-center su-flex-1 su-mx-auto su-max-w-[33rem] md:su-max-w-500 lg:su-max-w-800 su-px-30 md:su-px-38 2xl:su-px-[6.6rem] su-rs-py-3"
+            >
+              <FAIcon
```
Comment (iamrentman) [inline components/multicolumn-fact-panel/Component.jsx:61]: YESSSSS

File: packages/icons/FAIcon.jsx
Comment (iamrentman) [inline packages/icons/FAIcon.jsx:1]: 👏

**Review by iamrentman — APPROVED** [review]
Looks great to me Yvonne! Thanks again for your thoughtful work.

**Comment by yvonnetangsu** [comment]
> Looks great to me Yvonne! Thanks again for your thoughtful work.

Thank you for your review, @iamrentman! 😄

**Comment by yvonnetangsu** [comment]
I'm going to merge to `master,` add the component to PROD set then update the PROD git bridge 😄

**Pull request merged**